trulens_eval/trulens_eval/feedback/groundtruth.py

```diff
@@ -335,3 +345,277 @@ def rouge(
             ret = np.nan
 
         return ret
+
+
+class GroundTruthAggregator(WithClassInfo, SerialModel):
+    model_config: ClassVar[dict] = dict(
+        arbitrary_types_allowed=True, extra="allow"
+    )
+    """Aggregate benchmarking metrics for ground-truth-based evaluation on feedback fuctions."""
+
+    true_labels: List[int]
```

Comment:
can you align this arg name to the one in the GroundTruth class? I'm okay with either (ground_truth or true_labels)

Reply:
I've addressed the redundant `ground_truth` attribute in TruBenchmarkExperiment.

But to clarify, `ground_truth` has different meaning than true_labels in our current implementation.

`ground_truth` is eq to the golden set or ground truth data collection, and `true_labels` are just the labels or often time one of the columns in the GT dataset table (we don't have the schema defined yet but soon we will).